### DISCUSSION TITLE
[V4] Feature Requests

### DISCUSSION THREAD

**Discussion by saade** [saade/filament-fullcalendar#263] [General]
- [x] Remove default form schema and actions. Maybe we can provide a trait for BC
- [x] Fix misaligned "Today" button
- [ ] this is not done yet...

**Comment by mgkimsal** [comment]
Provide options for 'read only' calendar.

**Comment by krv** [comment]
Support for draggable
https://fullcalendar.io/docs/external-dragging

**Comment by xcalabuig** [comment]
Support for multi month grid
https://fullcalendar.io/docs/multimonth-grid

**Comment by medfrk** [comment]
Adding a way to add filter to the calendar like a form on the top

**Comment by danielreales7** [comment]
Add [https://fullcalendar.io/docs/scrollToTime](https://fullcalendar.io/docs/scrollToTime)

**Comment by edpalima** [comment]
How can I add color booked on calendar depends on status?

**Comment by agencetwogether** [comment]
Offer a possibility to show events of many models

**Comment by Danielopes7** [comment]
Provide synchronize with google calendar https://fullcalendar.io/docs/google-calendar

**Comment by SebastianBotez** [comment]
Hello. Is there any trick to make the calendar header sticky in resource timeline?
Tried with 
```
'height' => 'auto',
'stickyHeaderDates' => 'auto',
```
Thanks

**Comment by MV-ICT** [comment]
Enable HTML in the title

**Comment by sprtk-ches** [comment]
Support for resources https://fullcalendar.io/docs/resource-data

I'm willing to work on a PR for this if you think it's worth it.

EDIT: Oh shit, it seems there is one already 👍

**Comment by mathnabit** [comment]
Support for Day-Cell Render Hooks and datesSet

**Comment by eliotdwf** [comment]
Support for viewDidMount render hook (https://fullcalendar.io/docs/view-render-hooks)

**Comment by agencetwogether** [comment]
Support for Filament V4

**Comment by hussamitani** [comment]
Custom duration or at least a 3-day duration. (would be useful for yesterday, today, tomorrow or today, tomorrow, after tomorrow) <3

https://fullcalendar.io/docs/daygrid-view#custom-duration

**Comment by bilogic** [comment]
Filament 4?

**Comment by krekas** [comment]
@saade how's it going with v4 version? It's been a year with no news. What about filament v4 support?

**Comment by francescomugnai** [comment]
+1

**Comment by francescomugnai** [comment]
Any update on V4? We’d really need it for our next steps, so even a rough idea of the timeline would be great

**Comment by eugenefvdm** [comment]
@francescomugnai and @krekas ,

This is open source so you're welcome to do a pull request, run the Filament upgrade script, and reporting your findings.

**Comment by krekas** [comment]
@eugenefvdm sure for the filament v4 support. But it's been a year since this issue, and there are no updates. At least something could be said.
Also, there are issues with no response

**Comment by francescomugnai** [comment]
@eugenefvdm, I get the open source aspect. Still, this isn’t really about features or bug fixes, it’s just that the issue has been open for a year and even a small update would be really helpful for the community

**Comment by saade** [comment]
Thanks everyone for the patience and for sharing your feedback.

I know this issue has been open for quite a while, so let me clarify where things stand:
- Filament v4 support – this is still on the roadmap. I haven’t been able to dedicate as much time to it as I’d like, which is why updates have been sparse. The upgrade is not just a quick migration, since a few things in the calendar integration need to be revisited.
- Community PRs – as always, contributions are welcome. If anyone wants to run the Filament upgrade script and share findings in a PR (even draft), that would help move things forward more quickly.
- Timeline – I don’t have an exact ETA to share yet, but the plan is to start testing compatibility with v4 soon. Even partial feedback from the community on what’s working / what’s broken will help shape the release.

In the meantime, if you’re blocked by specific issues with Filament v4, feel free to open smaller, focused issues so they can be tracked separately.

Thanks again for keeping this thread active and for helping shape the next release.

**Comment by chadidi** [comment]
Any updates on when the Filament V4 support gonna be released?

**Comment by saade** [comment]
V4 BETA support is on the way :)

**Comment by jhoanborges** [comment]
+1

**Comment by MuhammadKazimSadiq** [comment]
where has the progress reached for v4?

**Comment by saade** [comment]
V4 is already available, but not all features are done.

**Reply by MuhammadKazimSadiq** [reply]
I wasn't able to install it in a new laravel 12 project...

**Reply by krekas** [reply]
Was able to install in a new laravel 12 project. v4 is in beta, keep that in mind.
